src/tests/ftest/util/verify_perms.py

```diff
@@ -0,0 +1,441 @@
+#!/usr/bin/env python3
+"""
+  (C) Copyright 2022 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+import sys
+import os
+import getpass
+from argparse import ArgumentParser
+from multiprocessing import Process, Queue  # MUST be a multiprocessing Queue
+import traceback
+import tempfile
+from itertools import product
+import logging
+from functools import partial
+
+from logger_utils import get_console_handler
+from user_utils import get_user_uid_gid
+from run_utils import run_local, RunException
+
+# Set up a logger for the console messages
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+logger.addHandler(get_console_handler("%(message)s", logging.INFO))
+
+
+class VerifyPermsError(Exception):
+    '''Base Exception class'''
+
+
+def create(path, entry_type, owner=None):
+    '''Create an entry.
+
+    Args:
+        path (str): path to create
+        entry_type (str): file or dir
+        owner (str, optional): user to create as. Defaults to current user
+
+    Raises:
+        ValueError: on invalid input
+
+    '''
+    if entry_type not in ('file', 'dir'):
+        raise ValueError(f'Invalid entry_type: {entry_type}')
+    if os.path.exists(path):
+        raise ValueError(f'Path already exists: {path}')
+    logger.info('Creating %s %s', entry_type, path)
+    if owner:
+        owner_uid_gid = get_user_uid_gid(owner)
+        _as_user(owner_uid_gid, _create, path, entry_type)
+    else:
+        _create(path, entry_type)
+
+
+def _create(path, entry_type):
```

Comment:
You might want to add method description.

Reply:
Done, thanks